src/ParametricOptInterface.jl

```diff
@@ -70,6 +72,105 @@ mutable struct ParametricOptimizer{T, OT <: MOI.ModelLike} <: MOI.AbstractOptimi
     end
 end
 
+function MOI.is_empty(model::ParametricOptimizer)
+    return MOI.is_empty(model.optimizer) &&
+    isempty(model.parameters) &&
+    isempty(model.parameters_name) &&
+    isempty(model.variables) &&
+    isempty(model.updated_parameters) &&
+    isempty(model.variables) &&
+    model.last_index_added == 0 &&
+    isempty(model.affine_constraint_cache) &&
+    isempty(model.quadratic_constraint_cache_pv) &&
+    isempty(model.quadratic_constraint_cache_pp) &&
+    isempty(model.quadratic_constraint_cache_pc) &&
+    isempty(model.quadratic_constraint_variables_associated_to_parameters_cache) &&
+    isempty(model.quadratic_added_cache) &&
+    model.last_quad_add_added == 0 &&
+    isempty(model.affine_objective_cache) &&
+    isempty(model.quadratic_objective_cache_pv) &&
+    isempty(model.quadratic_objective_cache_pp) &&
+    isempty(model.quadratic_objective_cache_pc) &&
+    isempty(model.quadratic_objective_variables_associated_to_parameters_cache)
+end
+
+function MOI.supports_constraint(
+    model::ParametricOptimizer,
+    F::Type{<:MOI.AbstractFunction},
+    S::Type{<:MOI.AbstractSet})
+
+    return MOI.supports_constraint(model.optimizer, F, S)
+end
+
+function MOI.supports(
+    model::ParametricOptimizer,
+    attr::Union{MOI.ObjectiveSense,
+            MOI.ObjectiveFunction{MOI.SingleVariable},
+            MOI.ObjectiveFunction{MOI.ScalarAffineFunction{T}},
+            }) where T
+    return MOI.supports(model.optimizer, attr)
+end
+
+function MOI.empty!(model::ParametricOptimizer{T}) where T
+    MOI.empty!(model.optimizer)
+    model.parameters = Dict{MOI.VariableIndex, Float64}()
```

Comment:
```suggestion
    empty!(model.parameters)
```

You should be able to do this for the rest of the `Dicts.

Reply:
Well noticed, I will change it